podpac/core/pipeline/pipeline.py

```diff
@@ -5,12 +5,12 @@
 from __future__ import division, unicode_literals, print_function, absolute_import
 
 from collections import OrderedDict
-import json
+import json as json
```

Comment:
??? `import json` then rename as `json`?

Reply:
Oops, I'll fix that tomorrow real quick.